app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java

```diff
@@ -44,22 +44,22 @@
 
     @Test
     public void incrementButtonPressed_textIsUpdated() {
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("0")));
-        onView(withId(R.id.increment_bt)).perform(click());
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("1")));
+        onView(withId(R.id.step2_value)).check(matches(withText("0")));
```

Comment:
Of course there's no need, but AFAIK there's no rule that says they shouldn't. :)